src/main/java/io/github/antishake/AntiShake.java

```diff
@@ -257,27 +261,32 @@ void convolve(ArrayList<Double> impulseResponseSamples, ArrayList<Coordinate> ac
 
   /**
    * Returns the value of the given key from the config.properties file
+   * TODO Write a util class that returns value by type and default value
    *
    * @param key
    * @return value
    */
-  private static String getPropertyValue(String key) {
-    return properties.getProperty(key);
+  private static String getPropertyValue(String key, String defaultValue) {
+    if (properties.containsKey(key)) {
+      return properties.getProperty(key);
+    } else {
+      return defaultValue;
+    }
   }
 
   /**
    * Loads all the properties from config.properties file and assigns to
    * appropriate static variables
    */
   private static void loadProperties() {
-    SPRING_CONSTANT = Double.parseDouble(getPropertyValue("SPRING_CONSTANT"));
-    CIRCULAR_BUFFER_IN_SEC = Double.parseDouble(getPropertyValue("CIRCULAR_BUFFER_IN_SEC"));
-    SAMPLING_RATE_IN_HZ = Double.parseDouble(getPropertyValue("SAMPLING_RATE_IN_HZ"));
+    SPRING_CONSTANT = Double.parseDouble(getPropertyValue("SPRING_CONSTANT", "10"));
```

Comment:
If the property file will have an empty value, then parsing it to Double gonna throw error. Rather than just parsing it to Double directly, its better to check if its empty or not. 

Unless we are creating extra Strings, it will be fine.